docs/index.rst

```diff
@@ -11,6 +11,7 @@ Table of content
    :maxdepth: 1
 
    overview
+   kinto-admin
```

Comment:
I think kinto-admin isn't that relevant to be in the second position of the documentation. I would probably put in the tutorial section, IDK.